OpenRA.Mods.Common/Scripting/Properties/CaptureProperties.cs

```diff
@@ -39,11 +39,12 @@ public void Capture(Actor target)
 
 			if (capturable != null)
 			{
-				if (captures.Any(x => !x.IsTraitDisabled && x.Info.CaptureTypes.Contains(capturable.Type)))
-				{
-					Self.QueueActivity(new CaptureActor(Self, target));
-					return;
-				}
+				foreach (var c in captures)
```

Comment:
Style nit: if an inner block has braces then the outer block should also have braces.